tests/Emitter/SapiEmitterTest.php

```diff
@@ -64,11 +145,11 @@ public function shouldNotOutputBodyIfEmitToldSo(): void
     public function contentLengthShouldNotBeOverwrittenIfPresent(): void
     {
         $length = 100;
-        $response = new Response(200, ['Content-length' => $length, 'X-Test' => 1], '');
+        $response = $this->createResponse(200, ['Content-Length' => $length, 'X-Test' => 1], '');
 
-        (new SapiEmitter())->emit($response);
+        $this->createEmitter()->emit($response);
 
-        $this->assertEquals(200, http_response_code());
+        $this->checkResponseCodeEquals(200);
```

Comment:
I suggests:
```php
$this->assertSame(200, $this->getResponseCode());
$this->assertSame(['x-test'...], $this->getResponseHeaders());
```

Reply:
`$this->assertSame(['x-test'...], $this->getResponseHeaders());`
This code does not take into account that the headers may be in different order

Reply:
```php
$this->assertEquals(['a' => 'b', 'b' => 'a'], ['b' => 'a', 'a' => 'b']);  // this works fine
```

Reply:
> ```
> $this->assertEquals(['a' => 'b', 'b' => 'a'], ['b' => 'a', 'a' => 'b']);  // this works fine
> ```

`$this->assertEquals([1,2,3], [3,2,1]); // suicide`

Reply:
How about multiple assertions with `assertContains()`?